Sources/Core/Shared/Logging.swift

```diff
@@ -266,14 +307,8 @@ extension NSOperation {
      `NSBlockOperation` types, it will return a 
      plain reading description.
     */
-    public var operationName: String {
-        get {
-            let _name = name ?? self.description
-            guard !_name.containsString("BlockOperation") else {
-                return "Unnamed Block Operation"
-            }
-            return _name
-        }
+    var operationName: String {
```

Comment:
Since this no longer filters for `"BlockOperation"`, I think the comment should be rewritten.